Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -54,14 +54,14 @@ final class TestDiscoveryCommand: CustomLLBuildCommand {
     ) throws {
         let stream = try LocalFileOutputByteStream(path)
 
-        let testsByClass = Dictionary(grouping: tests, by: { $0.name })
+        let testsByClassName = Dictionary(grouping: tests, by: { $0.name })
+        let classNames = testsByClassName.keys.sorted()
```

Comment:
Just a thought, but would it make sense to sort the entries themselves by key and hold onto that?
```
let classNamesAndTests = testsByClassName.sorted(by: { $0.key < $1.key })
```
That would avoid the `foo[bar]!` in the first loop, instead using just `.value`, and the second loop below could just use `.key`?  WDYT?

Reply:
good idea, fixed